code/game/objects/structures/watercloset.dm

```diff
@@ -200,7 +200,7 @@
 	icon_state = "urinalcake"
 	item_state = "urinalcake"
 	w_class = WEIGHT_CLASS_TINY
-	list_reagents = list(/datum/reagent/chlorine = 3, /datum/reagent/ammonia = 1)
+	list_reagents = list(/datum/reagent/chlorine = 10, /datum/reagent/ammonia = 5)
 	foodtype = TOXIC | GROSS
```

Comment:
urinal cakes are yummy not gross fix this injustice

Reply:
I TRIED, THAT GAVE A COMPILE ERROR SO I REVERTED IT

Reply:
![image](https://github.com/yogstation13/Yogstation/assets/108117184/85a02a12-52a4-4322-94dc-7bc51b4cca4f)
SEE